packages/cli/src/commands/ui/create/atomic.ts

```diff
@@ -24,7 +24,7 @@ interface AtomicArguments {
 
 export default class Atomic extends Command {
   public static cliPackage = '@coveo/create-atomic';
-  public static requiredNodeVersion = '>=14.0.0';
+  public static requiredNodeVersion = '>=14.0.0 <17.0.0'; // https://github.com/netlify/cli/issues/3617
```

Comment:
👁️

Reply:
Still gotta test with Stencil, we don't have the webpack dev server anymore... I'll update the required version if it fails

Reply:
Tried it with node 14 and now it works 😌

Reply:
feelsgoodman